firmware/projects/Demo/CANErrors/errors.dbc

```diff
@@ -0,0 +1,9 @@
+VERSION ""
+
+BS_:
+
+BU_: TMS
+
+// Defines one 64 bit signal where each bit represents a different error
+BO_ 390 TMS_ERROR: 8 TMS
+    SG_ Errors : 0|64@1- (1,0) [0|0] "bitfield" Vector__XXX
```

Comment:
this generates an `int64_t` but we're using it for the bit value so `uint64_t` is better. Check the DBC syntax for how to specify an unsigned value